src/components/Person/Person.jsx

```diff
@@ -1 +1,14 @@
-// export const Person = ({ person }) => ();
+export const Person = ({ person }) => {
+  const { name, age, sex, isMarried, partnerName } = person;
+  const partner = sex === 'm' ? 'wife' : 'husband';
+
+  return (
+    <section className="Person">
+      <h2 className="Person__name">My name is {name}</h2>
+      {age ? <p className="Person__age">I am {age}</p> : null}
```

Comment:
Here you can improve code by using logical AND `&&` instead of ternary operator with `null`. 
This will make the code a little more compact.

```suggestion
      {age && <p className="Person__age">I am {age}</p>}
```